integr8ly/keycloak-operator
# Adding necessary support for OpenShift 4 IDP plugin

## Motivation
In order to support OpenShift 4 cluster as IDP for a realm, we need to use plugin on older versions of keycloak.

## What
Adding volume, volume mounts and postStart script to copy theme files for the new plugin.

## Verification Steps
Accompanying plugins init container is updated in this PR: https://github.com/integr8ly/sso_plugins_init/pull/2, it will need to be merged or you will need to edit image tag to use "quay.io/integreatly/sso_plugins_init:add-openshift-4-idp"
Create keycloak instance with "openshift4-idp" in plugins list (just as in updated examples).
Login into admin console and go to "Identity Providers" page.
Drop-down for new provider should contain "Openshift v4" provider, choose it.
In the next step you should see a form like this:
![Screenshot 2019-07-19 at 14 55 00](https://user-images.githubusercontent.com/1605799/61536715-93225480-aa35-11e9-8b66-eeb0ebd51389.png)


## Checklist:
